scheduler/src/cook/mesos/api.clj

```diff
@@ -1629,9 +1635,18 @@
                          override-group-immutability? (boolean (get params :override-group-immutability))
                          pool-name (get params :pool)
                          pool (when pool-name (d/entity (d/db conn) [:pool/name pool-name]))
-                         uuid->count (pc/map-vals count (group-by :uuid jobs))]
+                         uuid->count (pc/map-vals count (group-by :uuid jobs))
+                         time-until-out-of-debt (rt/time-until-out-of-debt-millis! job-submission-rate-limiter user)
+                         in-debt? (not (zero? time-until-out-of-debt))]
                      (try
+                       (when in-debt?
+                         (log/info (str "User " user " is inserting too quickly (will be out of debt in "
+                                        (/ time-until-out-of-debt 1000.0) " seconds)")))
                        (cond
+                         (and in-debt? (rt/enforce? job-submission-rate-limiter))
+                         [true {::error (str "User " user " is inserting too quickly. Not allowed to insert for "
+                                             (/ time-until-out-of-debt 1000.0) " seconds")}]
```

Comment:
Should this message match the one that gets logged?

Reply:
No. One is telling a user why their response was rejected. Within the context of a user, it says directly that they won't be able to insert for a while, and says when, with no reference to tokens, buckets or anything else.

The other is informative, and tells us someone is in debt, in language we understand. I don't want it to say that a user had their submisison rejected, because if we were not enforcing, it wasn't rejected.